tools/.depend

```diff
@@ -160,6 +160,8 @@ ocamlmklib.cmo : \
 ocamlmklib.cmx : \
     ../utils/misc.cmx \
     ../utils/config.cmx
+ocamlmklibconfig.cmo :
+ocamlmklibconfig.cmx :
```

Comment:
Small-to-tiny nit - your worktree has an old `tools/ocamlmklibconfig.ml` file knocking around

Reply:
Thanks! I was wondering what was causing alldepend to differ on that machine.

Reply:
It's a nuisance that it picks up all files - I often up with `foo.cmx` (or ruder basenames, depending on how frustrating the bug is...) getting picked up, but I don't expect it's worth making `alldepend` more complicated than it already is!